src/natcap/invest/globio.py

```diff
@@ -943,22 +950,21 @@ def _collapse_infrastructure_op(*infrastructure_array_list):
         # necessarily nodata
         infrastructure_result = numpy.zeros(
             infrastructure_array_list[0].shape, dtype=numpy.uint8)
-
-        nodata_mask = numpy.isclose(
-            infrastructure_array_list[0], infrastructure_nodata_list[0])
-
-        infrastructure_mask = (infrastructure_array_list[0] > 0) & ~nodata_mask
-
-        for index in range(1, len(infrastructure_array_list)):
-            current_nodata = numpy.isclose(
-                infrastructure_array_list[index],
-                infrastructure_nodata_list[index])
-
-            infrastructure_mask = (
-                infrastructure_mask |
-                ((infrastructure_array_list[index] > 0) & ~current_nodata))
-
-            nodata_mask = (nodata_mask & current_nodata)
+        
+        nodata_mask = numpy.full(infrastructure_array_list[0].shape, True)
+        infrastructure_mask = numpy.full(infrastructure_array_list[0].shape, False)
+
+        for index in range(0, len(infrastructure_array_list)):
```

Comment:
This is a pretty trivial change, but since we're starting from index 0 (the default), this can be abbreviated to `range(len(infrastructure_array_list))` if you like to achieve the same effect.